sql/core/src/test/resources/sql-tests/inputs/ansi/parse-schema-string.sql

```diff
@@ -0,0 +1,5 @@
+-- Use keywords as attribute names
```

Comment:
let's put this file in `.../inputs/` and then import it here, so that it's run with annsi on and off.

Reply:
done